middleware/http.py

```diff
@@ -4,18 +4,24 @@
 from django.http import HttpResponseForbidden
 from django.template import RequestContext, Template, TemplateDoesNotExist, loader
 
-
 """
 # Middleware to allow the display of a 403.html template when a
 # 403 error is raised.
 """
 
 
+try:
+    from django.utils.deprecation import MiddlewareMixin
+except ImportError:
```

Comment:
This shouldn't be necessary since we don't support multiple Django versions.

Reply:
I'm a bit unsure of how to do it without doing this.

I guess we would have to rewrite the middleware to fit the new type?
https://docs.djangoproject.com/en/2.1/topics/http/middleware/#upgrading-middleware

Reply:
Rewriting it is probably a good idea, but what I actually meant was that the `try import-here except ImportError` code shouldn't be necessary. 
This kind of code is usually used in libraries that need to support multiple versions of Django and therefore have to provide a fallback in case the import is not included. 
Since we know that the `MiddlewareMixin` will be there we don't have to wrap it in `try except`.

Reply:
Ah, I misunderstood a bit there, should be fixed now.

Reply:
Would it be OK to just go for this solution here, and set up an issue for refactoring it?

Reply:
Absolutely! This PR is already big enough.